src/App.tsx

```diff
@@ -3,12 +3,17 @@ import './App.css';
 import OAuth2Form from './OAuth2Form';
 import AppInitializer from './AppInitializer'
 import Versions from './Versions'
+import { SynapseContextConsumer, SynapseContextType } from 'synapse-react-client/dist/utils/SynapseContext';
```

Comment:
New way of passing the current access token around.  Instead of using the react context, we have a SynapseContextProvider (and consumer, for old cases like this where we can't use the react hook)